website/snippets/_new-sl-setup.md

```diff
@@ -13,32 +13,41 @@ Select the environment where you want to enable the Semantic Layer:
 
-:::tip dbt Cloud Enterprise can skip to [Add more credentials](#4-add-more-credentials)
-dbt Cloud Enterprise plans can add multiple credentials and have a different set up. Skip to [Add more credentials](#4-add-more-credentials) for more configuration details.
-:::
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-select-env.jpg" width="75%" title="Select the deployment environment to run your Semantic Layer against."/>
 
 ### 2. Add a credential and create service tokens
 
-The dbt Semantic Layer uses [service tokens](/docs/dbt-cloud-apis/service-tokens) for authentication which are tied to an underlying data platform credential that you configure. The credential configured is used to execute queries that the Semantic Layer issues against your data platform. This credential controls the physical access to underlying data accessed by the Semantic Layer, and all access policies set in the data platform for this credential will be respected.
+The dbt Semantic Layer uses [service tokens](/docs/dbt-cloud-apis/service-tokens) for authentication which are tied to an underlying data platform credential that you configure. The credential configured is used to execute queries that the Semantic Layer issues against your data platform. 
+
+This credential controls the physical access to underlying data accessed by the Semantic Layer, and all access policies set in the data platform for this credential will be respected.
+
+| Feature | Team plan | Enterprise plan |
+| --- | :---: | :---: |
+| Service tokens | Can create multiple service tokens linked to one credential. | Can use multiple credentials and link multiple service tokens to each credential. Note that you cannot link a single service token to more than one credential. |
+| Credentials per project | One credential per project. | Can [add multiple](#4-add-more-credentials) credentials per project. |
+| Link multiple service tokens to a single credential | ✅ | ✅ |
 
-dbt Cloud Enterprise plans can add multiple credentials and map those to service tokens. Refer to [Add more credentials](#4-add-more-credentials) for more information.
+*If you're on a Team plan and need to add more credentials, consider upgrading to our [Enterprise plan](https://www.getdbt.com/contact). Enterprise users can refer to [Add more credentials](#4-add-more-credentials) for detailed steps on adding multiple credentials.*
 
-1. In the **Set Up Semantic Layer Configuration** page, enter the credentials specific to your data platform that you want the Semantic Layer to use.
+1. After selecting the deployment environment, you should see the **Credentials & service tokens** page. 
+2. Click the **Add Semantic Layer credential** button. 
+3. In the **1. Add credentials** section, enter the credentials specific to your data platform that you want the Semantic Layer to use.
```

Comment:
```suggestion
3. In the **Add credentials** section, enter the credentials specific to your data platform that you want the Semantic Layer to use.
```

Reply:
Hiya @mirnawong1 

I haven't committed this as I was't sure if the addition of the number 1 was intentional. This is a non-blocker btw.

Kind Regards

Reply:
yes it's intentional bc it's par of the cloud ui flow and I'm referring to it. great question!